apps/web/.env.example

```
@@ -1,3 +1,5 @@
 NEXT_PUBLIC_STELLAR_NETWORK=
 NEXT_PUBLIC_USDC_ISSUER_TESTNET=
-NEXT_PUBLIC_USDC_ISSUER_MAINNET=
\ No newline at end of file
+NEXT_PUBLIC_USDC_ISSUER_MAINNET=
+NEXT_PUBLIC_SUPABASE_URL=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_URL key should go before the NEXT_PUBLIC_USDC_ISSUER_MAINNET key

(UnorderedKey)